services/docker-compose.devel.yml

```diff
@@ -111,26 +111,31 @@ services:
 
   webserver:
     volumes:
-      &webserver-volumes-dev
+      &webserver_volumes_devel
       - ./web/server:/devel/services/web/server
       - ../packages:/devel/packages
     environment:
-      &webserver-environment-dev
+      &webserver_environment_devel
       WEBSERVER_REMOTE_DEBUGGING_PORT: 3000
       SC_BOOT_MODE: debug
       WEBSERVER_LOGLEVEL: ${LOG_LEVEL:-DEBUG}
       WEBSERVER_PROFILING: ${WEBSERVER_PROFILING}
 
 
+  web-api:
```

Comment:
2 Questions here: 
1. all the other webserver-based services are prefixed with wb. why change it just for this one?
2. web API sounds like "a Web API is an application programming interface for either a web server or a web browser". This is confusing, at least for me as such a web API sounds a lot like something dedicated to the frontend which it is not. Suggestions: ```wb-api-server``` or something in that direction. I'm also ok if you want to rename all the ```wb-``` services.

Reply:
ok